Django_connected/connected_house/server/forms.py~

```diff
@@ -0,0 +1,91 @@
+from django import forms
```

Comment:
We actually don't need that file at all, it should not have been indexed as the .gitignore should prevent that